docker/README.md

```diff
@@ -82,11 +84,10 @@ To build the TorchServe image for a CPU device using the `master` branch, use th
 ./build_image.sh
 ```
 
-Alternatively, you can use following direct command- 
+Alternatively, you can use following direct command, (assuming you have followed steps in [Clone serve source](#first-things-first))- 
 ```bash 
-Make sure you are inside serve/docker and use following commands
-1. git clone https://github.com/pytorch/serve.git
-2. cd serve;git checkout <branch>;cd docker
+1. Do one more clone -> `git clone https://github.com/pytorch/serve.git`
```

Comment:
Why not do `ln -s $PWD/.. ./`

Reply:
Doesn't seem to work with the symlink `ln -s $PWD/.. ./serve`
```
 => ERROR [stage-0  6/14] ADD serve serve                                                                                                                              0.0s
------
 > [stage-0  6/14] ADD serve serve:
------
failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = failed to build LLB: failed to compute cache key: "/tmp/serve" not found: not found
```